extruct/rdfa.py

```diff
@@ -35,6 +38,105 @@
 
 class RDFaExtractor(object):
 
+    def _replaceNS(self, prop, html_element, head_element):
```

Comment:
I'm curious if there are any helpers in ``rdflib`` which could be used instead?

Reply:
I looked for it, but I couldn't find anything. I was able to find the initial context, which I have used already.

Reply:
I see, thanks for checking 👍